contracts/src/emergency/sequencer_uptime_feed.cairo

```diff
@@ -1,5 +1,10 @@
 #[contract]
 mod SequencerUptimeFeed {
+    use starknet::EthAddress;
+    use starknet::EthAddressSerde;
+    use starknet::EthAddressIntoFelt252;
+    use starknet::Felt252TryIntoEthAddress;
+    use starknet::EthAddressZeroable;
```

Comment:
I think trait imports are no longer required but we can test that out afterwards